src/OmniSharp.Host/Services/OmniSharpEnvironment.cs

```diff
@@ -38,6 +40,20 @@ public class OmniSharpEnvironment : IOmniSharpEnvironment
             TraceType = traceType;
             TransportType = transportType;
             OtherArgs = otherArgs;
+
+            // On Windows: %APPDATA%\OmniSharp\omnisharp.json
```

Comment:
No idea. On my machine, I see the following under `USERPROFILE`:

* .dnx
* .node-gyp
* .nuget
* .templateengine
* .vscode
* .vscode-insiders
* .vscode-oss-dev

I think we're in good company in picking `USERPROFILE`. :smile: